polybar-scripts/network-huawei-modem/network-huawei-modem.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*- # PEP 263
+
+from huawei_lte_api.Client import Client
+from huawei_lte_api.Connection import Connection
+import huawei
+
+
+MODEM_URL = f"http://{huawei.username}:{huawei.password}@{huawei.host}/"
```

Comment:
Would it make sense to add the configuration to the same file? So that you only have to manage a single script?

Reply:
Sure, if you like the passwords to be leaked to github =) Separate file with 600 rights this is my whim, but you can do whatever.

Reply:
fixed!